TestProjects/UniversalUpgradeTest/Packages/manifest.json

```diff
@@ -0,0 +1,52 @@
+{
+  "dependencies": {
+    "nuget.moq": "1.0.0",
+    "com.unity.ide.rider": "3.0.4",
+    "com.unity.ide.visualstudio": "2.0.7",
+    "com.unity.ide.vscode": "1.2.3",
+    "com.unity.testing.urp-upgrade": "file:../../../com.unity.testing.urp-upgrade",
```

Comment:
We should be able to safely remove this test project now if we can adjust the yamato job to to pack all package dependencies. @Unity-Technologies/gfx-sdets Please let us know how this can be done.